file/index.js

```diff
@@ -1,120 +1,120 @@
-
-    // Process files one by one
-    for (var index = 0; index < modifiedFiles.length; index++) {
-        let modifiedFile = modifiedFiles[index];
-
-        let params = {
-            Bucket: bucketName,
-            Key: modifiedFile.Key
-        };
-
-        let getObjectPromise = s3.getObject(params).promise();
-
-        getObjectPromise.then(function (result) {
-            let fileData = result.Body.toString('utf-8');
-            var rows=fileData.split("\n");
-            var headers;
-            var count=0;
-            
-            for(var i in rows){
-                if(rows[i].trim().length===0)continue;
-                count++;
-                var cols=rows[i].split("\t");
-                // Extract headers
-                if(count===1){
-                    headers=cols;
-                    continue;
-                }
-                
-                var obj={};
-                for(var j in cols){
-                    obj[headers[j]]=cols[j];
-                }
-                if (modifiedFile.Key.endsWith("_delete.tsv")) {
-                    response.delete.near_earth_objects.push(obj);
-                } else{
-                    response.insert.near_earth_objects.push(obj);
-                }
-            }
-        }).catch(function (err) {
-            callback(err); // Return when some error occurred while reading any file
-            console.log("Error in getting object : " + err + "\n");
-        });
-
-        await getObjectPromise;
-
-        // Same last modified of processed file so we process files after that in next run
-        response.state.since = modifiedFile.LastModified;
-    }
+    // Same last modified of processed file so we process files after that in next run
+    response.state.since = modifiedFile.LastModified;
+  }
 
-    // Once response in generated use callback to finish lambda execution with response
-    callback(null, response);
+  // Once response in generated use callback to finish lambda execution with response
+  onError(null, response);
 }
 
 function initializeResponse(state) {
-    // Don't assign the value directly, it means you are assigning as a reference, 
-    // Now state of response doesn't get affected when change state variable in method
-    return {
-        "state": {...state}, 
-        "insert": {
-            "near_earth_objects": []
-        },
-        "delete": {
-            "near_earth_objects": []
-        },
-        "schema": {
-            "near_earth_objects": {
-                "primary_key": ["neo_reference_id"]
-            }
-        },
-        hasMore: false
-    };
-}
\ No newline at end of file
+  // Don't assign the value directly, it means you are assigning as a reference, 
+  // Now state of response doesn't get affected when change state variable in method
+  return {
+    state: { ...state
+    },
+    insert: {
+      "near_earth_objects": []
```

Comment:
same for these field names (remove quotes), we need it only for `delete` as reserved word.